src/main/java/rs/hybridit/model/BookCopy.java

```diff
@@ -0,0 +1,79 @@
+package rs.hybridit.model;
+
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import rs.hybridit.dto.BookCopyDto;
+
+@Entity
+@Table(catalog = "dbhybridlibrary", name = "book_copies")
+public class BookCopy {
+
+	@Id
+	@GeneratedValue
+	private Long id;
+
+	private Date rentStart;
+
+	private Date rentEnd;
+
+	@ManyToOne(fetch = FetchType.EAGER)
+	private Book book;
+
+	public BookCopy() {
+	}
+
+	public BookCopy(Long id, Date rentStart, Date rent_end, Book book) {
+		this.id = id;
+		this.rentStart = rentStart;
+		this.rentEnd = rentEnd;
```

Comment:
you are assigning value of variable to itself, since you have used snake case rent_end as argument. Btw. don't use snake_case but camelCase in java code